File: Linguagem C Exercicios/C-Sequencial/TERRENO.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double largura,comprimento,area,valorMetro,valorTerreno = 0;

    printf("Digite a largura do terreno: ");
    scanf("%lf",&largura);
    printf("Digite o comprimento do terreno: ");
    scanf("%lf",&comprimento);
    printf("Preco por metro quadrado: R$ ");
    scanf("%lf",&valorMetro);

    area = largura*comprimento;
    printf("A area desse terreno e de: %.2lfm\n",area);

    valorTerreno = area * valorMetro;
    printf("O valor desse terreno e de: R$ %.2lf",valorTerreno);

    return 0;
}

File: Linguagem Java/Exercicios/Terreno.java
import java.util.Locale;
import java.util.Scanner;

public class Terreno {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double larg,comp,precoMetro,precoTotal,area;
		
		System.out.print("Digite a largura do terreno: ");
		larg = sc.nextDouble();
		System.out.print("Digite o comprimento do terreno: ");
		comp = sc.nextDouble();
		System.out.print("Digite o valor do metro quadrado: ");
		precoMetro = sc.nextDouble();
		
		area = larg*comp;
		precoTotal = area * precoMetro;
		
		System.out.println("Area do terreno: "+String.format("%.2f", area));
		System.out.println("Preco do terreno: "+String.format("%.2f", precoTotal));
		
		sc.close();

	}

}
